fog/view/enclave/impl/src/lib.rs

```diff
@@ -350,10 +360,33 @@ where
         )
     }
 
+    // Takes each MultiViewStoreResponseQuery
     fn get_minimum_highest_processed_block_count(
-        &self,
-        _shard_query_responses: Vec<(QueryResponse, BlockRange)>,
+        mut responses: Vec<DecryptedMultiViewStoreQueryResponse>,
     ) -> u64 {
-        todo!()
+        responses.sort_by(|a, b| a.block_range.start_block.cmp(&b.block_range.start_block));
```

Comment:
Maximum effort version:
```suggestion
        responses.sort_unstable_by_key(|resp| resp.block_range.start_block);
```

Reply:
Done.